### PR TITLE
Ret 3992 - Populate manual hearings object with values from caseData

### PR DESCRIPTION
**Pull request by lvo700**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-3992

### Change description ###

Map data onto the object created in [RET-3634](https://tools.hmcts.net/jira/browse/RET-3634) to be used in [RET-3635](https://tools.hmcts.net/jira/browse/RET-3635) when ExUI POSTs to /serviceHearingValues

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/et-hearings-api/values.yaml [hunk 1]
```diff
@@ -4,10 +4,10 @@ java:
   ingressHost: et-hearings-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: et
   keyVaults:
-    et-hearings-api:
-      secrets:
-        - name: et-hearings-api-s2s-secret
-          alias: ET_HEARINGS_S2S_SECRET
+#    et-hearings-api:
```
Comment (marcjudge) [inline charts/et-hearings-api/values.yaml:7]: Is this intentional?

File: src/main/java/uk/gov/hmcts/reform/et/helper/mapping/EmployeeObjectMapper.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Converts case data that is stored in a format used in http calls and wraps it in a class for API use.
+ */
+@Slf4j
+@Service
+public final class EmployeeObjectMapper {
```
Comment (marcjudge) [inline src/main/java/uk/gov/hmcts/reform/et/helper/mapping/EmployeeObjectMapper.java:19]: This is a bad name which I think I originally created in et-sya-api when I didn't fully understand the project. I think this should be renamed CaseDataMapper or something similar

File: charts/et-hearings-api/values.yaml (hunk 1 above)
Comment (brendanmchugh) [reply charts/et-hearings-api/values.yaml:7]: shouldn't need to comment out these, the latest updates to ecm-ccd-docker should allow master to work without doing that.

**Review by brendanmchugh — CHANGES_REQUESTED** [review]

File: charts/et-hearings-api/values.yaml (hunk 1 above)
Comment (lvo700) [reply charts/et-hearings-api/values.yaml]: Reverted

File: src/main/java/uk/gov/hmcts/reform/et/helper/mapping/EmployeeObjectMapper.java (hunk 2 above)
Comment (lvo700) [reply src/main/java/uk/gov/hmcts/reform/et/helper/mapping/EmployeeObjectMapper.java]: Renamed

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -48,20 +48,25 @@ azure:
...
   s2s-auth:
     url: ${SERVICE_AUTH_PROVIDER_URL:http://localhost:4502}
-    secret: ${ET_HEARINGS_S2S_SECRET:AAAAAAAAAAAAAAAA}
-    microservice: ${MICRO_SERVICE:et_hearings_api} 
+    secret: ${ET_HEARINGS_S2S_SECRET:AAAAAAAAAAAAAAAC}
+    microservice: ${MICRO_SERVICE:ccd_gw}
+  s2s-authorised:
+      services: ${IDAM_S2S_AUTH_MICROSERVICES:xui_webapp}
+
  # s2s-authorised:
```
[the remark this answers is not in the document]
Comment (lvo700) [reply src/main/resources/application.yaml:59]: Reverted

**Review by brendanmchugh — APPROVED** [review]

**Review by marcjudge — APPROVED** [review]

**Pull request merged**